tests/runtime/variable

```diff
@@ -121,3 +121,13 @@ EXPECT Attaching 1 probe...
 NAME variable declaration not initialized
 PROG BEGIN { let $a; if (false) { $a = 1; } @b = $a; exit();}
 EXPECT @b: 0
+
+NAME variable doesn't escape scope
+RUN {{BPFTRACE}} runtime/scripts/variable_scope.bt
+EXPECT 0
+
+NAME late variable declaration
+RUN {{BPFTRACE}} runtime/scripts/variable_scope_late_decl.bt
+EXPECT 1
+EXPECT 2
+EXPECT 0
```

Comment:
Unsure if ordering is guaranteed here. Maybe you wanna do it all in one line with `\n` separating values?

Reply:
I'm going to pair the `$x` variable with a string in a tuple to confirm they are coming from the expected places e.g.
```
EXPECT (a, 1)
EXPECT (b, 2)
EXPECT (c, 0)
```